### PR TITLE
Internal: Editor V2 - Icons package [ED-9855]

### PR DESCRIPTION
**Pull request by rotemee**

### CODE REVIEW

File: core/editor/config-providers/editor-v2-config-provider.php [hunk 1]
```diff
@@ -25,6 +25,7 @@ class Editor_V2_Config_Provider implements Config_Provider_Interface {
 		'ui',
 		'v1-adapters',
 		'store',
+		'icons',
```
Comment (StyleShit) [inline core/editor/config-providers/editor-v2-config-provider.php:28]: A-Z

File: packages/packages/icons/src/components/settings-icon.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { SvgIcon, SvgIconProps } from '@elementor/ui';
+
+const Settings = React.forwardRef( ( props: SvgIconProps, ref ) => {
```
Comment (StyleShit) [inline packages/packages/icons/src/components/settings-icon.tsx:4]:
```suggestion
const SettingsIcon = React.forwardRef( ( props: SvgIconProps, ref ) => {
```

File: packages/packages/icons/src/components/wide-screen-icon.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { SvgIcon, SvgIconProps } from '@elementor/ui';
+
+const WideScreenIcon = React.forwardRef( ( props: SvgIconProps, ref ) => {
```
Comment (StyleShit) [inline packages/packages/icons/src/components/wide-screen-icon.tsx:4]: https://dictionary.cambridge.org/dictionary/english/widescreen

it's a single word 🤷

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: core/editor/config-providers/editor-v2-config-provider.php (hunk 1 above)
Comment (rotemee) [reply core/editor/config-providers/editor-v2-config-provider.php]: Done.

File: packages/packages/icons/src/components/settings-icon.tsx (hunk 2 above)
Comment (rotemee) [reply packages/packages/icons/src/components/settings-icon.tsx]: Done.

File: packages/packages/icons/src/components/wide-screen-icon.tsx (hunk 3 above)
Comment (rotemee) [reply packages/packages/icons/src/components/wide-screen-icon.tsx]: Done.

**Review by StyleShit — DISMISSED** [review]

File: packages/packages/documents/src/components/top-bar/settings-button.tsx [hunk 4]
```diff
@@ -22,8 +22,9 @@ export default function SettingsButton( { type }: Props ) {
 				selected={ isActive }
 				disabled={ isBlocked }
 				onChange={ () => openRoute( 'panel/page-settings/settings' ) }
+				size="small"
 			>
-				<CogIcon />
+				<SettingsIcon />
```
Comment (Nevoss) [inline packages/packages/documents/src/components/top-bar/settings-button.tsx:27]: I think cog is the name in all the icons libs

**Review by Nevoss — APPROVED** [review]

**Pull request merged**